### PR TITLE
update rubocop-rspec v1.20 to remove `rubocop:disable RSpec/SubjectStub`

### PR DESCRIPTION
**Pull request by walf443**
rubocop-rspec v1.20 was released.

This release includes https://github.com/backus/rubocop-rspec/issues/488 fix.

-----------------

Before submitting the PR make sure the following are checked:

* [x] Wrote [good commit messages][1].
* [x] Commit message starts with `[Fix #issue-number]` (if the related issue exists).
* [x] Used the same coding conventions as the rest of the project.
* [x] Feature branch is up-to-date with `master` (if not - rebase it).
* [x] Squashed related commits together.
* [ ] Added tests.
* [x] Added an entry to the [Changelog](../blob/master/CHANGELOG.md) if the new code introduces user-observable changes. See [changelog entry format](../blob/master/CONTRIBUTING.md#changelog-entry-format).
* [ ] All tests(`rake spec`) are passing.
* [x] The new code doesn't generate RuboCop offenses that are checked by `rake internal_investigation`.
* [x] The PR relates to *only* one subject with a clear title
  and description in grammatically correct, complete sentences.
* [x] Updated cop documentation with `rake generate_cops_documentation` (required only when you've added a new cop or changed the configuration/documentation of an existing cop).

[1]: http://chris.beams.io/posts/git-commit/

### CODE REVIEW

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -32,7 +32,13 @@ RSpec/AnyInstance:
     - 'spec/rubocop/cop/lint/duplicate_methods_spec.rb'
     - 'spec/rubocop/target_finder_spec.rb'
 
-# Offense count: 2574
```
Comment (walf443) [inline .rubocop_todo.yml]: Is it better to wrote .rubocop.yml?

File: .rubocop_todo.yml [hunk 2]
```diff
@@ -32,7 +32,13 @@ RSpec/AnyInstance:
...
+# Offense count: 1004
+# Configuration parameters: Prefixes.
+# Prefixes: when, with
+RSpec/ContextWording:
```
Comment (walf443) [inline .rubocop_todo.yml]: Is it better to wrote .rubocop.yml?

Comment (Drenmi) [reply .rubocop_todo.yml]: It's okay to keep it here. We have a lot of legacy in our specs, but at some point I hope we'll catch up. 🙂

**Comment by bbatsov** [comment]
Thanks!

**Pull request merged**